tests/filter.json

```diff
@@ -466,6 +466,45 @@
         {"d": "f"}
       ]
     },
+    {
+      "name": "non-singluar existence, wildcard",
```

Comment:
Typo. "singular" rather than "singluar". Same goes for the following three test cases too.